python/fledge/common/microservice_management_client/microservice_management_client.py

```diff
@@ -86,8 +87,7 @@ def unregister_service(self, microservice_id):
         try:
             response["id"]
         except (KeyError, Exception) as ex:
-            _logger.exception("Could not unregister the micro-service having uuid %s, Reason: %s",
-                              microservice_id, str(ex))
+            _logger.exception(ex, "Could not unregister the micro-service having uuid {}".format(microservice_id))
```

Comment:
All good. Though I see we are using microservice, micro-service, service at different places for the same thing. uuid -> UUID